copula.html

```diff
@@ -10,83 +10,98 @@
-                     patches : [{offset : 0x4F8FD, off: [0x48], on : [0x90]}]
-                    },
-                    {
-                     name : "Unlock All Songs",
-                     patches : [{offset : 0x4FD84, off: [0x74, 0x16], on : [0x90, 0x90]}]
-                    },
-                    {
-                     name : "Level 12 Unlocked",
-                     patches : [{offset : 0x4FE35, off: [0x83, 0xFF, 0x02, 0x74, 0x0B, 0x83, 0xFF, 0x05, 0x74, 0x06, 0xB0, 0x01],
-                                                   on : [0x90, 0x90, 0x90, 0x90, 0x90, 0x90, 0x90, 0x90, 0x90, 0x90, 0x90, 0x90]}]
-                    },
-                    {
-                     name : "Cursor lock",
-                     tooltip : "After song finishes, song select remains on previous song",
-                     patches : [{offset : 0x63722, off: [0x74, 0x23], on : [0x90, 0x90]}]
-                    },
-                    {
-                     name : "CS-Style Song Start Delay",
-                     tooltip : "Holding Start will pause the song at the beginning until you release it",
-                     patches : [{offset : 0x6F3FF, off: [0x7C], on : [0xEB]}]
-                    },
-                    {
-                     name : "Disable HCN Tutorial",
-                     tooltip : "Don't show tutorial for Hell Charge Notes",
-                     patches : [{offset : 0x6F3E6, off: [0x5E, 0xE9, 0x24, 0xFF, 0xFF, 0xFF], on : [0x00, 0x00, 0x00, 0x00, 0x00, 0x00]}]
-                    },
-                    {
-                     name : "Quick Retry",
-                     tooltip : "Hold VEFX and Effect during a song to restart",
-                     patches : [{offset : 0x46724, off: [0x8A, 0xC3], on : [0xB0, 0x01]}]
-                    },
-                    {
-                     name : "Dark Mode",
-                     patches : [{offset : 0x68833, off: [0x74, 0x32], on : [0x90, 0x90]}]
-                    },
-                    {
-                     name : "Volume Bug Fix",
-                     tooltip : "If your volume gets forced to max, turn this on",
-                     patches : [{offset : 0xCB2B9, off: [0x00], on : [0x01]}]
-                    },
-                    {
-                     type : "union",
-                     name : "FREE PLAY text",
-                     offset : 0x1379A,
-                     patches : [
-                         {
-                          name : "On",
-                          patch : [0x88, 0x3D, 0x11, 0x10],
-                         },
-                         {
-                          name : "Off",
-                          patch : [0x87, 0x3D, 0x11, 0x10],
-                         },
-                         {
-                          name : "Replace with song name",
-                          patch : [0x54, 0x1D, 0x42, 0x11],
-                         },
-                     ]
-                    },
+                new DllPatcherContainer([
```

Comment:
This is still just one game and shouldn't be in a patch container

Reply:
Oh, I've been putting single games in patch containers in case a different version gets released

Reply:
If you've been doing this and I haven't noticed, I shouldn't call it out here then.